cni/pkg/repair/netns.go

```diff
@@ -112,7 +112,8 @@ func getPodNetNs(pod *corev1.Pod) (string, error) {
 			return err
 		})
 		if err != nil {
-			log.Warnf("failed to check proc %d netns interfaces: %v", p.PID, err)
+			// It is expected this will fail for all but one of the procs, so keep this at debug level
+			log.Debugf("failed to check proc %d netns interfaces: %v", p.PID, err)
```

Comment:
```suggestion
			log.Tracef("failed to check proc %d netns interfaces: %v", p.PID, err)
```

honestly, maybe Trace? it's kinda useless. Up to you tho.

Reply:
I wish -- https://github.com/istio/istio/issues/43030 :-(

Reply:
ill just drop it

Reply:
(my bad - I thought we had `trace` :sweat_smile: )